src/api/forms/FormsAPI.js

```diff
@@ -162,26 +162,16 @@ export default class FormsAPI extends EventEmitter {
             }
         }
 
-        function onFormDismiss(dismiss) {
+        function onFormAction(callback) {
             return () => {
                 if (element) {
                     formElement.remove();
                 } else {
                     overlay.dismiss();
                 }
 
-                if (dismiss) {
-                    dismiss();
-                }
-            };
-        }
-
-        function onFormSave(save) {
-            return () => {
-                overlay.dismiss();
-
-                if (save) {
-                    save(changes);
+                if (callback) {
+                    callback();
                 }
```

Comment:
Is saving not a default? Do we need to pass in a save callback anywhere now?

Reply:
The `onFormAction` handles both save and cancel cases, and dismisses the modal or removes the form element appropriately.